generators/app/templates/_gulpfile.js

```diff
@@ -8,9 +8,12 @@ var fs = require('fs');
 // config
 gulp.paths = {
   dist: 'www',
-  jsFiles: ['app/**/*.js', '!app/bower_components/**/*.js'],
+  jsFiles: ['app/**/*.js', '!app/bower_components/**/*.js', '!app/**/*.spec.js'],
```

Comment:
This is unrelated to jade, i will removed it, but it is related to an another problem when jsFiles contain also test js files, and then these files are bundled... aren't they?